packages/api-headless-cms/src/content/plugins/utils/findEntries.ts

```diff
@@ -52,6 +52,15 @@ export default async function findEntries<T = CmsContext>({
         query.locale = context.cms.locale.id;
     }
 
+    // For `manage` API we always want to multiple `limit` by available locales
```

Comment:
"multiple" => "multiply" :)

Reply:
BTW @Ashu96, please add a bit more description here so that it's clear why exactly we need this.